code/modules/surgery/bodyparts/hair.dm

```diff
@@ -60,7 +60,8 @@
 	hair_overlay = null
 	hair_gradient_overlay = null
 
-	hair_alpha = owner_species.hair_alpha
+	//hair_alpha = owner_species.hair_alpha // ORIGINAL
+	hair_alpha = human_head_owner.hair_alpha ? human_head_owner.hair_alpha : owner_species.hair_alpha // SKYRAT EDIT - Customization - Hair alpha
```

Comment:
For something so short, put the original on the same line, this make it take more space than it needs to, without helping with maintainability.

Reply:
That arguably makes it harder to read. Not going to lie.